.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,93 @@
+name: Bug report
+description: Submit a new bug report.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## This issue tracker is only for technical issues related to Monero.
+
+        * General monero questions and/or support requests should use Monero StackExchange at https://monero.stackexchange.com.
+        * For reporting security issues, please read instructions at https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md.
+
+
+        ----
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered. You can search [here](https://github.com/monero-project/monero/issues?q=is%3Aissue).
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: Tell us what went wrong
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: Tell us what you expected to happen
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Tell us how to reproduce your bug. Please attach related screenshots if necessary.
+        * Run-time or compile-time configuration options
+        * Actions taken
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: |
+        Please copy and paste any relevant log output or attach a debug log file.
+
+        You can get log information via --log-level=2 flag or by taking a look at log files at ~/.bitmonero/ in cli version. Or you can look at the log information in Settings->Log section in Monero-GUI.
```

Comment:
Should we leave this in? For example the wallet log file is in a different place than the daemon log file, also --log-level=2 affects the log that gets printed and written to the disk, so that "or" doesn't really make sense.

Reply:
I added a more detailed comment here in new push. Please don't hesitate to let me know if it needs to change again. Thanks